index.bs

```diff
@@ -922,6 +931,12 @@ optionally evidence of [=user consent=] to a specific transaction.
         authenticators are eligible to participate in a {{CredentialsContainer/create()}} or {{CredentialsContainer/get()}}
         operation. See [[#attachment]] for a description of the attachment values and their meanings.
 
+    :   <dfn>requireResidentKey</dfn>
+    ::  This member describes the [=[RPS]=]' requirements regarding availability of the [=credential private keys=]. If the
+        parameter is set to <a>true</a>, the authenticator SHOULD create a [=resident key=] when creating a [=scoped credential=].
```

Comment:
It is not necessary for that algorithm to be written in this case. We are doing this everywhere in the spec when it comes to authenticator operation. The way the platform finds if the authenticator is available is actually by using the authenticatorGetInfo method, which is documented in the CTAP spec and hard-coded in new authenticators already. The method is not there today anyway because that's really an editorial issue. I am happy with spending sometime later to update that section to more precisely describe the CTAP spec. 
I propose we just pass in the bit for now and update the algorithm to be more precise later. That entire section is already outdated compared to the existing CTAP spec.